src/FixPressureAtNodeAlgorithm.C

```diff
@@ -113,6 +113,8 @@ FixPressureAtNodeAlgorithm::execute()
       }
     });
   });
+
+  eqSystem_->linsys_->free_coeff_applier(deviceCoeffApplier);
```

Comment:
Why is this necessary? Were we leaking memory?

Reply:
On the TpetraLinearSystem class, we had to change it so that it is no longer holding onto the host/device coeff appliers as its member variables. So, a new coeff applier is constructed on each request. Because of this, we had to introduce a way to properly deallocate coeff appliers when no longer needed. This is one of the changes that we will likely to revisit to find a better way to handle this.
Just as a note, `free_coeff_applier` is an empty function in the `LinearSystem.h`, so this line will not affect `HypreLinearSystem`.